src/components/views/elements/Field.tsx

```diff
@@ -36,7 +36,21 @@ export interface IValidateOpts {
     allowEmpty?: boolean;
 }
 
-interface IProps {
+export interface IInputValidationProps {
```

Comment:
let's move this out to its own file (plus all of its related interfaces) if we're going to export it please.